apps/docs/content/guides/platform/manage-your-usage/monthly-active-users-sso.mdx

```diff
@@ -0,0 +1,71 @@
+---
+id: 'manage-usage-monthly-active-users-sso'
+title: 'Manage Monthly Active SSO Users usage'
+---
+
+## What you are charged for
+You are charged for the number of distinct users who log in or refresh their token during the billing cycle using a SAML 2.0 compatible identity provider. Each unique user is counted only once per billing cycle, regardless of how many times they authenticate. These users are referred to as "SSO MAUs".
+
+## How charges are calculated
+You are charged by SSO MAU.
```

Comment:
Like commented on the other MAU metrics, think it makes sense to give an example of how a MAU is calculated (i.e. multiple customer requests)